README.md

```diff
@@ -137,6 +138,18 @@ For more detailed information and advanced usage, please refer to the following:
 
 For updating versions, please refer to the [**Release Notes**](docs/release-notes/release-notes-0-to-14.md).
 
+## Build the Project
+
+You can build the project using Maven:
+
+```shell
+mvn clean install -DskipTests -DskipFormatting
```

Comment:
(Question)

Why would we suggest to skip tests, are our local integration tests running by default?

Reply:
Mainly speed, if users want to quickly test a snapshot version they shouldn't need to wait for test success or formatting